Orange/widgets/evaluate/owparameterfitter.py

```diff
@@ -318,93 +368,150 @@ def _add_plot(self):
         box.layout().addWidget(self.graph)
 
     def _add_controls(self):
-        box = gui.vBox(self.controlArea, "Settings")
-        self.__combo = gui.comboBox(box, self, "parameter_index",
+        layout = QGridLayout()
+        gui.widgetBox(self.controlArea, "Settings", orientation=layout)
+        self.__combo = gui.comboBox(None, self, "parameter_index",
                                     model=self.__parameters_model,
                                     callback=self.__on_parameter_changed)
-
-        buttons = gui.radioButtons(box, self, "type",
-                                   callback=self.__on_setting_changed)
-
-        gui.appendRadioButton(buttons, "Range")
-        hbox = gui.indentedBox(buttons, 20, orientation=Qt.Horizontal)
-        kw = {"minv": -MIN_MAX_SPIN, "maxv": MIN_MAX_SPIN,
-              "callback": self.__on_setting_changed}
-        self.__spin_min = gui.spin(hbox, self, "minimum", label="Min:", **kw)
-        self.__spin_max = gui.spin(hbox, self, "maximum", label="Max:", **kw)
-
-        gui.appendRadioButton(buttons, "Manual")
-        hbox = gui.indentedBox(box, 20, orientation=Qt.Horizontal)
-        gui.lineEdit(hbox, self, "manual_steps", placeholderText="10, 20, 30",
-                     callback=self.__on_setting_changed)
-
-        box = gui.vBox(self.controlArea, "Steps preview")
-        self.preview = ""
-        gui.label(box, self, "%(preview)s", wordWrap=True)
+        layout.addWidget(self.__combo, 0, 0, 1, 2)
+
+        buttons = gui.radioButtons(None, self, "type",
+                                   callback=self.__on_type_changed)
+        button = gui.appendRadioButton(buttons, "Range:")
+        layout.addWidget(button, 1, 0)
+
+        kw = dict(minv=-MIN_MAX_SPIN, maxv=MIN_MAX_SPIN,
+                  alignment=Qt.AlignRight,
+                  callback=self.__on_min_max_changed)
+        box = gui.hBox(None)
+        self.__spin_min = gui.spin(box, self, "minimum", label="From:", **kw)
```

Comment:
This label is slightly to high. I'll take care of this if we decided to keep this layout.